daemon/algod/api/algod.oas2.json

```diff
@@ -2429,6 +2429,89 @@
           }
         }
       }
+    },
+    "/v2/devmode/blocks/offset": {
+      "get": {
+        "description": "Gets the current timestamp offset.",
+        "tags": [
```

Comment:
Should we create a new tag for dev mode only endpoints? Alternatively, put it behind a node config like `EnableDeveloperAPI`?

Currently, the node returns a `400` error after checking the genesis configs if the node is not in a dev mode network.

Reply:
I think it's fine as is. DevMode is already switched on via the genesis file so adding another place where people have to do it would feel wrong.